packages/server/src/app/handlers/handle-ongoing-process-action.ts

```diff
@@ -0,0 +1,95 @@
+import { ethers } from 'ethers';
+import { Signature } from 'fmg-core';
+import { channelID } from 'fmg-core/lib/channel';
+import { appAttributesFromBytes } from 'fmg-nitro-adjudicator';
+import { CommitmentReceived, communication, RelayableAction, StrategyProposed } from 'magmo-wallet';
+import { HUB_ADDRESS } from '../../constants';
+import { errors } from '../../wallet';
+import { getProcess } from '../../wallet/db/queries/walletProcess';
+import { updateLedgerChannel } from '../../wallet/services';
+import { Blockchain } from '../../wallet/services/blockchain';
+import { updateRPSChannel } from '../services/rpsChannelManager';
+
+export async function handleOngoingProcessAction(ctx) {
+  const action: RelayableAction = ctx.request.body;
+  switch (action.type) {
+    case 'WALLET.NEW_PROCESS.CONCLUDE_INSTIGATED':
+    case 'WALLET.FUNDING.STRATEGY_APPROVED':
+      return ctx;
+    case 'WALLET.COMMON.COMMITMENT_RECEIVED':
+      return handleCommitmentReceived(ctx, action);
+    case 'WALLET.FUNDING.STRATEGY_PROPOSED':
+      return handleStrategyProposed(ctx, action);
+  }
+}
+
+async function handleStrategyProposed(ctx, action: StrategyProposed) {
+  const { processId } = action;
+  const process = await getProcess(processId);
+  if (!process) {
+    throw errors.processMissing(processId);
```

Comment:
Do you know if the error is returned as part of the response to the client?

Reply:
Not in this case. I haven't quite figured out how to manage error handling, and wasn't concerned with it at the moment